app/views/products/new_from_rails_template.html.erb

```diff
@@ -0,0 +1,13 @@
+<%= render 'header' %>
+
+<%= bootstrap_form_for(@product, as: :product, url: create_from_rails_template_path(type: @type)) do |f| %>
+    <%= render "form_directory", f: f if @product.new_record? %>
```

Comment:
Will we ever go to new when `@product.new_record?` is not true? Could we just have `<%= render "form_directory" %>`?